eth/backend.go

```diff
@@ -549,8 +545,6 @@ func (s *Ethereum) Start() error {
 	if err != nil {
 		return err
 	}
-	// periodically flush databases
-	go s.syncDatabases()
```

Comment:
Could you add in the PR description how long it's been a no-op? Or even better, which PR/issue made it a no-op.